control-plane/subcommand/server-acl-init/command.go

```diff
@@ -666,58 +666,12 @@ func (c *Command) Run(args []string) int {
 	}
 
 	if len(c.flagTerminatingGatewayNames) > 0 {
-		// Create a token for each terminating gateway name. Each gateway needs a
```

Comment:
extracting this code into a private helper function so that it can also be used for upcoming ingress gateway work.